### DISCUSSION TITLE
typo in version number ? v1.4.2 returned from a 1.4.3 installed

### DISCUSSION THREAD

**Discussion by hvia** [rhash/RHash#213] [Q&A] [answered]
from RHash V1.4.3 compiled and installed, I get:
rhash --version
RHash v1.4.2
maybe a typo?

**Comment by rhash** [comment, marked as answer]
Yup, it's a typo. Use commit 01bcf3b6163a1e43845fbc08efe9ff551534db3c as a patch to fix it.
